client/service/src/lib.rs

```diff
@@ -73,6 +73,7 @@ pub use sc_chain_spec::{
 
 pub use sc_consensus::ImportQueue;
 pub use sc_executor::NativeExecutionDispatch;
+pub use sc_network_common::sync::warp::WarpSyncParams;
```

Comment:
It is a good patter to re-export types that people will require when interacting with your crate. Otherwise the user would need to import `sc-network-common` as well.